### DISCUSSION TITLE
Welcome!

### DISCUSSION THREAD

**Discussion by smarek** [OK-DMR/ok-dmrlib#2] [Announcements]
This is a place for:

- Any DMR related questions, tips and ideas
- Discussing tools, usages, approaches to problems
- Helping others in their radio (amateur and professional) adventures and endeavours

Strictly forbidden is:
- To share files, that are protected by NDAs or similar, which could harm the whole community
- To suggest or lead others to harmful or illegal activities

**Comment by alwinhb** [comment]
@smarek 

Do you have any technical system documentation for DMR apart from the ETSI, that looks a bit confusing. Also any reference for DMR Audio samples?

**Reply by smarek** [reply]
ETSI documentation is valid and constitutive for all DMR applications and deployments, yes it has specific form, but all the vendors and users must follow this documentation

Documentation on specific vendor, equipment, software is maybe what you seek? If so, feel free to start new discussion, this is Welcome post thread.

Also regarding audio, DMR from the start uses DVSI AMBE+2™ Vocoder @ 3.6 kbps ( https://www.dvsinc.com/soft_products/mobile_radio.shtml ), which is closed-source/closed-hardware solution. For more than a decade there has been reverse engineering effort resulting in various quality tools that allow to listen to voice from air/network captures. 

And last there is effort to provide Codec2 via firmware modification to some DMR mobile stations (handsets). It's not ETSI committee lead effort, so it won't be significant for critical/government/public usage anytime soon, but from technical perspective, if you need more insight, dig around, because they usually need to benchmark (compare) to existing vocoder implementation and QoS.

References:
- https://www.dvsinc.com/papers/iambe.html (history of IMBE / AMBE and some technical details) 
- https://news.ycombinator.com/item?id=19866595 (on ambe being non-optimal for non-english voice compression) 
- https://github.com/g4klx/AMBETools (last commit 2023)
- https://github.com/balr0g/imbe_vocoder (last commit 11 years ago)
